src/Sylius/Bundle/LocaleBundle/Templating/Helper/LocaleHelper.php

```diff
@@ -37,9 +37,9 @@ public function __construct(LocaleConverterInterface $localeConverter)
     /**
      * {@inheritdoc}
      */
-    public function convertCodeToName($localeCode)
+    public function convertCodeToName($localeCode, $locale = 'en')
     {
-        return $this->localeConverter->convertCodeToName($localeCode);
+        return $this->localeConverter->convertCodeToName($localeCode, $locale);
```

Comment:
Sorry about the composer.lock... I totally agree with your proposal ! I just need some advice on the specs, should I keep the original method "it_converts_locales_code_to_name" ? I was thinking about removing it and renaming the one added here in "it_converts_locales_code_to_name_with_provided_locale_or_default_locale". WDYT ?